src/Components/ExerciseInputs.js

```diff
@@ -0,0 +1,114 @@
+import { useState } from 'react';
+import SpaceBetween from './Styles/SpaceBetween';
+import Button from './Styles/Button';
+import styled from 'styled-components';
+
+export default function ExerciseInputs() {
+  const [exerciseInputs, setExerciseInputs] = useState([
+    { Exercise: '', Weight: '', Repetitions: '', Sets: '' },
```

Comment:
I would strongly recommend to use lowercase instead of capital case letter for object keys.
```suggestion
    { exercise: '', weight: '', repetitions: '', sets: '' },
```